packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-modal.tsx

```diff
@@ -255,7 +255,7 @@ const JurisdictionModal = ({
                         toggleModal={toggleJurisdictionModal}
                         type='button'
                         context={context}
-                        height='664px'
+                        // height='664px'
```

Comment:
If it's not needed, you can remove it completely. Also, add sanity test on Jurisdiction modal to QA Checklist so they would check if it's not breaking anything in other conditions.